content/open-source/slsa/what-is-slsa.md

```diff
@@ -16,69 +16,61 @@ weight: 50
 toc: true
 ---
 
-SLSA (pronounced “salsa”), or Supply chain Levels for Software Artifacts, is a security framework consisting of standards and controls that prevent tampering, improve integrity, and secure packages and infrastructure. While cyberattacks like [SolarWinds](https://www.gao.gov/assets/gao-22-104746.pdf) and [Codecov](https://www.reuters.com/technology/codecov-hackers-breached-hundreds-restricted-customer-sites-sources-2021-04-19/) have demonstrated the importance of protecting software from tampering and malicious compromise, the complexity of the software development lifecycle can leave many feeling unable to adequately understand or respond to these specific security issues. 
+SLSA (pronounced “salsa”), or Supply-chain Levels for Software Artifacts, is a security framework consisting of standards and controls that prevent tampering, improve integrity, and secure packages and infrastructure. While cyberattacks like [SolarWinds](https://www.gao.gov/assets/gao-22-104746.pdf) and [Codecov](https://www.reuters.com/technology/codecov-hackers-breached-hundreds-restricted-customer-sites-sources-2021-04-19/) have demonstrated the importance of protecting software from tampering and malicious compromise, the complexity of the software development lifecycle can leave many feeling unable to adequately understand or respond to these specific security issues. 
```

Comment:
I was on the fence about that hyphen. The [official website](https://slsa.dev/#:~:text=Supply%2Dchain%20Levels%20for%20Software%20Artifacts%2C%20or%20SLSA%20(%22salsa%22).) uses it when introducing the acronym, so I followed suit, but other resources don't include it.

Reply:
Hmm, that makes it a little less clear cut for me, but I'd still err on the side of not having the hyphen. Mostly because I was distracted by why that occurrence had a hyphen but no other ones do.